srcpkgs/muse/template

```diff
@@ -0,0 +1,47 @@
+# Template file for 'muse'
+# TODO: add option for libinstpatch (don't have it packaged)
+pkgname=muse
+version=3.1.1
+revision=1
+build_style=cmake
+build_helper=qmake
+configure_args="-DENABLE_RTAUDIO=OFF -DENABLE_INSTPATCH=OFF $(vopt_bool alsa ENABLE_ALSA)
+ $(vopt_bool lash ENABLE_LASH) $(vopt_bool lrdf ENABLE_LRDF) $(vopt_bool osc ENABLE_OSC)
+ $(vopt_bool dssi ENABLE_DSSI) $(vopt_bool fluidsynth ENABLE_FLUID) $(vopt_bool lv2 ENABLE_LV2)
+ $(vopt_bool gtk ENABLE_LV2_GTK2) $(vopt_bool rubberband ENABLE_RUBBERBAND)
+ $(vopt_bool python ENABLE_PYTHON)"
+hostmakedepends="pkg-config"
+makedepends="jack-devel ladspa-sdk libsamplerate-devel libsndfile-devel
+ qt5-svg-devel qt5-tools-devel $(vopt_if alsa alsa-lib-devel) $(vopt_if lash ladish-devel)
+ $(vopt_if lrdf liblrdf-devel) $(vopt_if osc liblo-devel) $(vopt_if dssi dssi-devel)
+ $(vopt_if fluidsynth fluidsynth-devel) $(vopt_if lv2 lv2) $(vopt_if lv2 lilv-devel)
+ $(vopt_if gtk gtkmm2-devel) $(vopt_if rubberband rubberband-devel) $(vopt_if python python3-devel)"
+depends="desktop-file-utils qt5-svg $(vopt_if python python3-six) $(vopt_if python python3-Pyro4)"
+short_desc="MIDI/Audio sequencer with recording and editing capabilities"
+maintainer="Evgeny Ermakov <evgeny.v.ermakov@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://muse-sequencer.github.io/"
+_name="${pkgname}-${version}"
+distfiles="https://github.com/muse-sequencer/${pkgname}/releases/download/${_name//[-.]/_}/${_name}.tar.gz"
+checksum=d40126543d549af30090d1f98422588d74ad1e336ad43e911bd4b02eff217842
+python_version=3
+
+# Package build options
+build_options="alsa lash lrdf osc dssi fluidsynth lv2 gtk rubberband python"
+build_options_default=$build_options
+desc_option_alsa="Enable support for ALSA"
+desc_option_lash="Enable LASH session management system for JACK"
+desc_option_lrdf="Enable Lightweight RDF. Provides extra info for LADSPA plugins"
+desc_option_osc="Enable Lightweight Open Sound Control (DSSI also recommended)"
+desc_option_dssi="Enable Disposable Soft Synth Interface (OSC also needed)"
+desc_option_fluidsynth="Enable fluidsynth softsynth plugins"
+desc_option_lv2="Enable LV2 plugins and synths support"
+desc_option_gtk="Enable LV2 user interface GTK2 support"
+desc_option_rubberband="Enable RubberBand time-stretching and pitch-shifting library"
+desc_option_python="Enable support for Python"
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" qt5-host-tools qt5-qmake"
+fi
+
+configure_args+=" -DLIB_INSTALL_DIR=/usr/lib"
```

Comment:
This can go on top with the initial declaration of `configure_args`.

Reply:
indeed

Reply:
Done.